bumped to livekit-client v2 and react-components client v2

thesrs02 opened the pull request:
bumped up the live kit-client version to 2 along with react components for the known Chrome Issues. This was the update released by the live kit team on March 30th.

thesrs02 committed:
bumped to livekit-client v2 and react-components client v2

davidzhao commented:
@thesrs02 thank you for the PR. could you revert the deletion of README, package-lock.json, and .env.example?

thesrs02 committed:
readme added back with screenshot updated

thesrs02 committed:
.env.example added back

thesrs02 commented:
> davidzhao

Added readme back with screenshot updated along with the .env.example that I deleted for myself. As for the package.json file. There was an issue where even after updating the packages, the lock file was not updated so I had to delete and re-created it otherwise the upgrades wouldn't work. Something reported by multiple clients as well.

davidzhao committed:
remove spaces in .env.example

davidzhao committed:
Update presence.tsx
